app.js

```diff
@@ -1,33 +1,21 @@
 console.log("JS Loaded");
 
-
-
 $('#btn').click(function(){
  var value = $('#whichCrypto').val();
- var output = $('#outputValue');
+ var $output = $('#outputValue');
 $.ajax({
     url: "https://api.coindesk.com/v1/bpi/currentprice.json",
     method: "GET",
     success: function(response) {
       for (var currency in response.bpi) {
-  console.log(currency);
-  console.log(response.bpi[currency].rate_float);
 if (currency === value) {
-  output.text(currency + " " + "=" + " " + response.bpi[currency].rate_float)
-}
-
-
-
-
+  $output.text(`${currency} = ${response.bpi[currency].rate_float)}`;
 }
-
-      
+}   
     },
     error: function(info) {
-        console.log(info)
+        console.log('There was an error', info);
```

Comment:
If you just print out `info`, it won't be helpful to you as a developer. You could even be more detailed and say `"There was an error fetching the converted currency"` or something like that.